test/fail.test.js

```diff
@@ -72,11 +72,14 @@ Your **[semantic-release](https://github.com/semantic-release/semantic-release)*
       labels: "semantic-release",
       title: "The automated release is failing 🚨",
     })
-    .reply(200);
+    .reply(200, { id: 3, web_url: "https://gitlab.com/test_user/test_repo/-/issues/3" });
 
   await fail(pluginConfig, { env, options, branch, errors, logger: t.context.logger });
 
   t.true(gitlab.isDone());
+  t.assert(t.context.logger.log.getCall(0).args[0], "Created issue #%d: %s.");
+  t.true(t.context.logger.log.getCall(0).args[1] === 3);
+  t.assert(t.context.logger.log.getCall(0).args[2], "https://gitlab.com/test_user/test_repo/-/issues/3");
```

Comment:
Can we do this like the [other similar assertions](https://github.com/semantic-release/gitlab/blob/master/test/verify.test.js#L127)?

```suggestion
  t.assert(t.context.log.args[0], ["Created issue #%d: %s.", 3, "https://gitlab.com/test_user/test_repo/-/issues/3"]);
```

Reply:
`assert` requires a string for comparison, at least this is the error I receive: `"The assertion message must be a string"`. I would change it to `deepEqual`:
```js
t.deepEqual(t.context.log.args[0], ["Created issue #%d: %s.", 3, "https://gitlab.com/test_user/test_repo/-/issues/3"]);
```
Similar to [here](https://github.com/semantic-release/gitlab/blob/master/test/verify.test.js#L107).
Is this ok?

Reply:
That's what I meant, thanks!